modules/troubleshooting-common-problems-with-distributed-workloads-for-administrators.adoc

```diff
@@ -0,0 +1,286 @@
+:_module-type: REFERENCE
+
+[id="troubleshooting-common-problems-with-distributed-workloads-for-administrators_{context}"]
+= Troubleshooting common problems with distributed workloads for administrators
+
+[role='_abstract']
+If your users are experiencing errors in {productname-long} relating to distributed workloads, read this section to understand what could be causing the problem, and how to resolve the problem.
+
+ifndef::upstream[]
+If you cannot see the problem here or in the release notes, contact {org-name} Support.
+endif::[]
+
+== A user's Ray cluster is in a suspended state
+
+.Problem
+The resource quota specified in the cluster queue configuration might be insufficient, or the resource flavor might not yet be created.
+
+.Diagnosis
+The user's Ray cluster head pod or worker pods remain in a suspended state. 
+Check the status of the `Workloads` resource that is created with the `RayCluster` resource.
+The `status.conditions.message` field provides the reason for the suspended state, as shown in the following example:
+
+[source,bash]
+----
+status:
+ conditions:
+   - lastTransitionTime: '2024-05-29T13:05:09Z'
+     message: 'couldn''t assign flavors to pod set small-group-jobtest12: insufficient quota for nvidia.com/gpu in flavor default-flavor in ClusterQueue'
+
+----
+
+.Resolution
+Verify that the resource flavor is created.
+Check the user's cluster queue configuration to ensure that the resources that they requested are within the limits defined for the project.
+If necessary, increase the resource quota. 
+
+ifndef::upstream[]
+For information about configuring resource flavors and quotas, see link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads#configuring-quota-management-for-distributed-workloads_distributed-workloads[Configuring quota management for distributed workloads].
+endif::[]
+ifdef::upstream[]
+For information about configuring resource flavors and quotas, see link:{odhdocshome}/working-with-distributed-workloads/#configuring-quota-management-for-distributed-workloads_distributed-workloads[Configuring quota management for distributed workloads].
```

Comment:
I'm also wondering if the provided links are supposed to work when clicked on from here. - Forgive my poor markdown/adoc knowledge.

Reply:
The attributes in the links will be populated in the published version. We'll check the links after publishing to make sure that they work.

No worries about your comments, better to ask in case we missed something! Thank you!

Reply:
That's perfect, thank you Breda!